contrib/connector/iscsi/helper.go

```diff
@@ -262,7 +270,7 @@ func connect(connMap map[string]interface{}) (string, error) {
 	log.Println("devicepath is ", devicePath)
 
 	// Discovery
-	err = discovery(portal)
+	_, err = discovery(portal)
 	if err != nil {
 		return "", err
 	}
```

Comment:
return nil, err

Reply:
It should be "", if the return type is string.